package.json

```diff
@@ -69,7 +76,6 @@
     "source-map-support": "^0.5.12",
     "ts-node": "^8.1.0",
     "tslint": "^5.15.0",
-    "tslint-config-airbnb": "^5.11.1",
-    "typescript": "^3.7.2"
+    "typescript": "^3.9.5"
```

Comment:
>The latest major version of @slack/bolt is supported to build against a minimum TypeScript version of v3.7.

We're saying that this library is built against 3.7+ in the document. https://slack.dev/bolt-js/tutorial/using-typescript

Is it possible to hold off upgrading the version?

Reply:
Yes, it's not necessary to upgrade the typescript version. I reverted it back to the original in this [commit](https://github.com/slackapi/bolt-js/pull/529/commits/0d538222344c41afa40ccf010e764e514a366354).